### ISSUE TITLE
SetPartialWords side effects

### ISSUE THREAD

**Issue by johngebbie** [alphacep/vosk-api#1224]
Hello, I just reporting something I found surprising.
I thought `SetPartialWords(1)` just made the output more verbose, but it significantly changes the "partial" field with the full (not per word) text.
Printing the "partial" field with `SetPartialWords` off, you see each word come in and the text rapidly changing, but with it on, you can get 4/5 words are a time.

I'm guessing it uses a different system when it's on and is not a bug, but thought it might be worth documenting it has side effects.
It took me a while to hunt down why my program wasn't working the same.

**Comment by nshmyrev** [comment]
Yes, it is true. In general partial confidence is a big problem which might need a different search algorithm.
